app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -74,4 +76,10 @@ def set_access_control_headers
     headers['Access-Control-Allow-Origin'] = '*'
   end
 
+  def ab_test_onboarding(community_id:)
+    if(rand(2) == 1)
+      FeatureFlagService::API::Api.features.enable(community_id: community_id, features: [:onboarding])
```

Comment:
this should probably have a more specific name, such as `onboarding_experiment_1`